graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/ConfigGenerator.java

```diff
@@ -0,0 +1,309 @@
+        }
+
+        ModulePartNode moduleNode = createModulePartNode(createEmptyNodeList(),
+                createNodeList(memberDeclarationNodes), createToken(EOF_TOKEN));
+        TextDocument textDocument = TextDocuments.from(EMPTY_STRING);
+        SyntaxTree configTypeSyntaxTree = SyntaxTree.from(textDocument);
+        return configTypeSyntaxTree.modifyWith(moduleNode);
+    }
+
+    private TypeDefinitionNode constructConnectionConfig(ModuleMemberDeclarationNode node, AuthConfig authConfig) {
+        RecordTypeDescriptorNode connectionConfigNode = (RecordTypeDescriptorNode)
+                ((TypeDefinitionNode) node).typeDescriptor();
+        MetadataNode authMetadataNode = getMetadataNode("Configurations related to client authentication");
+        StringBuilder authName = new StringBuilder(((AuthConfigType) authConfig.
+                getAuthConfigTypes().toArray()[0]).getValue());
+        for (int i = 1; i < authConfig.getAuthConfigTypes().size(); i++) {
+            authName.append("|").append(((AuthConfigType) authConfig.getAuthConfigTypes().
+                    toArray()[i]).getValue());
+        }
+        IdentifierToken authFieldName = AbstractNodeFactory.createIdentifierToken(
+                authName.toString());
+
+        Node authConfigNode = createRecordFieldNode(authMetadataNode, null, authFieldName,
+                createIdentifierToken("auth"), null, createToken(SEMICOLON_TOKEN));
+
+        List<Node> tokens = new ArrayList<>();
+        NodeList<Node> fields = connectionConfigNode.fields();
+        tokens.add(authConfigNode);
+        for (Node n : fields) {
+            tokens.add(n);
+        }
+        NodeList<Node> nodeList = createNodeList(tokens);
+        RecordTypeDescriptorNode.RecordTypeDescriptorNodeModifier recordTypeDescriptorNodeModifier =
+                connectionConfigNode.modify().withFields(nodeList);
+        connectionConfigNode = recordTypeDescriptorNodeModifier.apply();
+
+        return createTypeDefinitionNode(((TypeDefinitionNode) node).metadata().get(),
+                createToken(PUBLIC_KEYWORD), createToken(TYPE_KEYWORD), ((TypeDefinitionNode) node).typeName(),
+                connectionConfigNode, createToken(SEMICOLON_TOKEN));
+    }
+
+    /**
+     * Gets the path of the utils.bal template at the time of execution.
+     *
+     * @return Path to utils.bal file in the temporary directory created
+     * @throws IOException When failed to get the templates/utils.bal file from resources
+     */
+    private Path getResourceFilePath() throws IOException {
+        Path path = null;
+        ClassLoader classLoader = getClass().getClassLoader();
+        InputStream inputStream = classLoader.getResourceAsStream("templates/config_types.bal");
+        if (inputStream != null) {
+            String clientSyntaxTreeString = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+            Path tmpDir = Files.createTempDirectory(".config_types-tmp" + System.nanoTime());
```

Comment:
Here too let's try to use the word `graphql` to avoid conflicts

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/commit/eb969edf9f95622fe8d6dc7ecc000346998b403e